pkgs/wire/types.go

```diff
@@ -110,16 +110,11 @@ type Exchange struct {
 	Commits []byte `ssz-max:"2048"`
 }
 
-type Ping struct {
-	// Operators involved in the DKG
-	Operators []*spec.Operator `ssz-max:"13"`
-	// Initiator public key
-	InitiatorPublicKey []byte `ssz-max:"2048"`
-}
-
 type Pong struct {
-	ID     uint64
-	PubKey []byte `ssz-max:"2048"`
+	ID                 uint64
+	PubKey             []byte `ssz-max:"2048"`
+	Multisig           bool
+	EthClientConnected bool
```

Comment:
we check connection to eth client when start operator, if no it cant start, but we can loose it after we started. So we can be `Multisig=true` and `EthClientConnected=false`.
For old operator versions we dont have both fields so it will be `false`, `false`.
If we have only EthClientConnected then we cant distinguish between old and new version, only comparing `version`. So `Multisig` is rather informational for users not to compare versions, as old will be always `false`